resources/eventing/charts/publisher-proxy/templates/peerauthentication.yaml

```diff
@@ -9,4 +9,4 @@ spec:
     matchLabels: {{- include "publisher-proxy.peerauth.selectorLabels" . | nindent 6 }}
   portLevelMtls:
     {{ .Values.metrics.config.port }}:
-      mode: STRICT
\ No newline at end of file
+      mode: PERMISSIVE
```

Comment:
Is there a way to make only the port `9090` as `PERMISSIVE`?

Reply:
This change results in:
```
portLevelMtls:
    "9090":
         mode: PERMISSIVE
```
according to [istio docs](https://istio.io/latest/docs/reference/config/security/peer_authentication/), the permissive option will only apply to the 9090 port, all the other ports will remain in the default(which is strict).